maboss/simulation.py

```diff
@@ -336,3 +343,15 @@ def copy_and_update_parameters(sim, parameters):
     new_sim.update_parameters( **parameters )
     return new_sim
 
+def to_biolqm(maboss_model):
+    
+    path = tempfile.mkdtemp()
+    bnet_fd, bnet_filename = tempfile.mkstemp(dir=path, suffix='.bnet')
+    os.close(bnet_fd)
+
+    with open(bnet_filename, "w") as bnet_file:
+        for node, rule in maboss_model.get_logical_rules().items():
+            bnet_file.write("%s, %s\n" % (node, rule))
+
+    return biolqm.load(bnet_filename)
+
```

Comment:
You can also do a similar thing to convert to minibn:
```
def to_minibn(maboss_model):
    from colomoto import minibn # minibn is not (yet) a proper colomoto tool
    ... output to .bnet file
    return minibn.BooleanNetwork.load(bnet_filename)
```

Reply:
Done